build/automation/var/profile/local.mk

```diff
@@ -9,7 +9,8 @@ ALLOWED_ORIGINS := *
 # Datastore URLs
 ELASTICSEARCH_URL := elasticsearch.sfs.local:9200
 #POSTCODE_LOCATION_DYNAMO_URL := https://dynamodb.eu-west-2.amazonaws.com
-POSTCODE_LOCATION_DYNAMO_URL := http://localhost:8000/
+#POSTCODE_LOCATION_DYNAMO_URL := http://localhost:8000/
```

Comment:
This will conflict with master when you come to merge. Please make sure you take the master changes.